src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,28 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.13.0
+TBD
+
+### Features
+- Add Access Control to [Volume Snapshots](/docs/enterprise/administration/volume-snapshots/)
+- Add link to Community [Community](https://community.okteto.com) in help menu.
+- Inject OKTETO_NAMESPACE and OKTETO_DOMAIN environment variables in every pod
+- Remove Namespace limit on licenses
+
+### Improvements
+- Upgrade Okteto CLI to [2.3.3](https://github.com/okteto/okteto/releases/tag/2.3.3)
+- Upgrade registry to [2.8.1](https://github.com/distribution/distribution/releases/tag/v2.8.1)
+- Better deduplication in secrets query. Prioritize user secrets over cluster secrets.
+- Handle PVC update in the mutating webhook. Fixes Okteto CLI issue [#2599](https://github.com/okteto/okteto/issues/2599)
```

Comment:
Shouldn't this be in bugfixes?

Reply:
The bug is in the cli not the backend. I don't think this is fixing an issue in OE itself but rather an improvement that fixes a cli issue